docs/source/03_tutorial/05_namespacing_pipelines.md

```diff
@@ -0,0 +1,230 @@
+# Namespacing pipelines
+
+This section covers the following:
+
+* A brief introduction to namespaces and modular pipelines
+* How to convert the existing spaceflights project into a namespaced one
+* This pattern is viewed as the modern way to use Kedro after being first introduced in version 0.17.0
+
+Adding namespaces to [modular pipelines](https://kedro.readthedocs.io/en/stable/06_nodes_and_pipelines/03_modular_pipelines.html#modular-pipelines) unlocks some sophisticated functionality in Kedro
+
+1. You are able to [instantiate the same pipeline structure multiple times](https://kedro.readthedocs.io/en/stable/06_nodes_and_pipelines/03_modular_pipelines.html#how-to-use-a-modular-pipeline-twice), but provide different inputs/outputs.
+2. You can unlock the full power of [micro-packaging](https://kedro.readthedocs.io/en/stable/06_nodes_and_pipelines/03_modular_pipelines.html#how-to-share-a-modular-pipeline).
+3. You can de-clutter your mental model with Kedro-Viz rendering collapsible components.
+
+    ![collapsible](../meta/images/collapsible.gif)
+
+## Adding a namespace to the  `data_processing` pipeline
+
+* Update the code in `src/spaceflights_tutorial/pipelines/data_processing/pipeline.py` using the snippet below.
```

Comment:
Previously we assumed the project was called `kedro_tutorial` and now it's `spaceflights_tutorial`, copy pasting the code below will lead to import errors.